src/main/kotlin/no/nav/medlemskap/domene/arbeidsforhold/Arbeidsforhold.kt

```diff
@@ -171,6 +173,20 @@ data class Arbeidsforhold(
             return true
         }
 
+        private fun lovligAntallDagerBorte(ytelse: Ytelse, kontrollPeriode: Kontrollperiode, tillatDagersHullIPeriode: Long): Int {
+            when (ytelse) {
+                Ytelse.SYKEPENGER -> {
+                    // if kontrollperiode.isreferansePeriode >.
```

Comment:
kan fjernes

Reply:
Var det ikke slik at logikken for validering ville være forskjellig på sykepenger og i fremtiden dagpenger? i så fall må vi legge til denne koden igjen når vi skal validere andre ytelser i fremtiden?

Reply:
Tenkte bare på at selve kommentaren var overflødig nå

Reply:
selvsagt.. fikser!!